app/ui-react/packages/ui/stories/Integration/Editor/IntegrationEditorExtensionList.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import { action } from "@storybook/addon-actions";
+import { text } from "@storybook/addon-knobs";
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+
+const stories = storiesOf('Integration/Editor/IntegrationEditorExtensionList', module);
+
+import {
+  IntegrationEditorExtensionTable
+} from '../../../src';
```

Comment:
might be good to just import the one file, I seem to recall someone hitting an issue with an import done this way.

Reply:
WebStorm complains for the following:
`import { IntegrationEditorExtensionTable } from '../../../src/Integration/Editor/IntegrationEditorExtensionTable';`
but is okay with this:
`import { IntegrationEditorExtensionTable } from '../../../src/Integration/Editor';`
wdyt? I can also ignore it, seems to be the case for some of the other stories.